tests/UnifiedSpecTests/Constraint/Matches.php

```diff
@@ -256,7 +257,7 @@ private function assertMatchesOperator(BSONDocument $operator, $actual, string $
             $constraint = IsBsonType::anyOf(...(array) $operator['$$type']);
 
             if (! $constraint->evaluate($actual, '', true)) {
-                self::failAt(sprintf('%s is not an expected BSON type: %s', $this->exporter()->shortenedExport($actual), implode(', ', $types)), $keyPath);
```

Comment:
Noted that `$types` was never defined. This dates back to [4608a73](https://github.com/mongodb/mongo-php-library/commit/4608a734993be0b0aa1e0e838fa52e4caa9bfff2#diff-5911ac585872dc4a55ff2fe77af552d46e160b4d5636699c9cf2c1211217e25eR250). Curious that it wasn't caught by static analysis, unless this snuck into the baseline.